realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/mongodb/sync/SyncConfigurationTests.kt

```diff
@@ -163,7 +168,8 @@ class SyncConfigurationTests {
                     }
                 })
                 .build()
-        assertNotNull(config.initialDataTransaction)
+        // FIXME How to verify this without access to initialDataTransaction
+//        assertNotNull(config.initialDataTransaction)
```

Comment:
if it isn't public, it probably should be.

Reply:
There was actually a package protected access on `RealmConfiguration` so I just updraded it to `protected`. Could make it public if that makes sense, but to me it seems ok to leave non-public.